src/translations/fr/about.mdx

```diff
@@ -1,4 +1,4 @@
-Cela signifie que son code source peut être librement réutilisé et modifié, tant que les améliorations effectuées sont mises à disposition de la communauté sous les mêmes conditions. Cela signifie également que les décisions d'évolution du logiciel seront prises de manière de plus en plus démocratique au fur et à mesure que la communauté qui le construit s'agrandira.
+Cela signifie que son code source peut être librement réutilisé et modifié, tant que les améliorations effectuées sont mises à disposition de la communauté sous les mêmes conditions. Cela signifie également que les décisions d'évolution du logiciel seront prises de manière de plus en plus démocratique au fur et à mesure que la communauté qui le construit s'agrandira. Son [budget](/budget) est public et transparent.
```

Comment:
Same, use Link if you want to not reload the page

Reply:
Fixed with https://github.com/ambanum/OpenTermsArchive.org/pull/154/commits/c301b16dcc1a37a6d39f7282bcb39e146172b580

Reply:
You should use `<Link href="/budget"><a>budget</a></Link>`